index.js

```diff
@@ -153,21 +185,28 @@ function redisStore(args) {
    * @method get
    * @param {String} key - The cache key
    * @param {Object} [options] - The options (optional)
+   * @param {boolean|Object} options.gzip - gzip configuration
    * @param {Function} cb - A callback that returns a potential error and the response
    */
   self.get = function(key, options, cb) {
     if (typeof options === 'function') {
       cb = options;
+      options = {};
+    }
+    options.parse = true;
+
+    var gzip = (options.gzip || options.gzip === false) ? options.gzip : redisOptions.gzip;
```

Comment:
Unfortunately, no, if you want users to be able to turn gzip on/off at the set/get level. If `redisOptions.gzip` is truthy (object) and the user wants to turn it off for a particular set/get, right now they can pass `false`:

`myCache.set('myCacheKey', 'myCacheValue', { gzip: false }, function...`
`myCache.get('myCacheKey', { gzip: false }, function...`

The more simple comparison would not catch that. The `options.gzip` would evaluate to `false` and the `redisOptions.gzip` would be used. That's why I have an explicit test for `false`.  It's the same type of test that you are doing in the `set` method for the `options.ttl` value, in order to catch zero:

`var ttl = (options.ttl || options.ttl === 0) ? options.ttl : redisOptions.ttl;`

That's actually where I got the idea.   :)